docs/Configuration.md

```diff
@@ -4,18 +4,97 @@ title: Configuration
 permalink: /configuration/
 nav_order: 4
 ---
-# Configuration 
+
+# Configuration
 
 VS Code settings enable you to customize various aspects of its function. The InterSystems extensions provide settings used to configure VS Code for ObjectScript development.
 
+{: #code-configuration-basic}
+
+## Basic Configuration
+
+VS Code has a concept of a workspace, which is a set of directories you want to use when you're working on a particular project. In the simplest setup when you are working within a single directory, a VS Code workspace is just the root folder of your project. In this case you keep your settings in two files inside a `.vscode` directory located at the root of your project. Those two files are `settings.json`, which contains most configuration settings, and `launch.json`, which contains debugging configurations.
+
+Here is the simplest `settings.json` file contents:
+
+{: #code-workspace-simple}
+
+```json
+{
+    "objectscript.conn": {
```

Comment:
If the `server` key is present then you don't need the connection information.

Reply:
Moved the server key to a separate example.